theme/json.go

```diff
@@ -54,43 +54,50 @@ func fromJSONWithFallback(r io.Reader, fallback fyne.Theme) (fyne.Theme, error)
 		return fallback, err
 	}
 
-	return &jsonTheme{data: th, fallback: fallback, parsedColors: map[hexColor]color.Color{}}, nil
+	return &jsonTheme{data: th, fallback: fallback}, nil
 }
 
-type hexColor string
+type hexColor struct {
+	color color.Color
+}
```

Comment:
We can probably also just use color.Color directly and remove this struct. The name hexColor is also not technically correct any more as it isn't a string.